polling_stations/apps/file_uploads/templates/file_uploads/council_detail.html

```diff
@@ -47,6 +47,9 @@ <h2>Uploads</h2>
                         </td>
                         <td>
                             {{ upload.status_emoji }}
+                            {% if upload.status == "Pending" or "Waiting" %}
```

Comment:
I don't think this does what you want. I think it should be: `if upload.status == "Pending" or upload.status =="Waiting"`.